core/genesis_test.go

```diff
@@ -204,7 +209,7 @@ func TestStatefulPrecompilesConfigure(t *testing.T) {
 			}
 
 			db := rawdb.NewMemoryDatabase()
-			_, err := SetupGenesisBlock(db, genesis)
+			_, err := SetupGenesisBlock(db, genesis, common.Hash{})
```

Comment:
Should we use the genesis hash here? It looks like this still passes because it's setting up the database for the first time, so it never hits the point where it checks the config against the last accepted hash.

Reply:
done